cmd/reviewdog/main.go

```diff
@@ -164,6 +166,7 @@ func init() {
 	flag.BoolVar(&opt.guessPullRequest, "guess", false, guessPullRequestDoc)
 	flag.BoolVar(&opt.tee, "tee", false, teeDoc)
 	flag.Var(&opt.filterMode, "filter-mode", filterModeDoc)
+	flag.BoolVar(&opt.failOnError, "fail-on-error", false, failOnErrorDoc)
```

Comment:
I would like to see the information about using this flag in README.md

Reply:
I have added `Exit Codes` section with some explanations